com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/ca/lsp/cobol/service/utils/FileSystemUtils.java

```diff
@@ -165,6 +192,21 @@ public Path getPath(@Nonnull String basePath, @Nonnull String... more) {
         .collect(Collectors.toList());
   }
 
+  /**
+   * Creates a directory by creating all nonexistent parent directories first.
+   *
+   * @param path
+   * @return
+   */
+  public Path createFolders(Path path) {
+    try {
+      return Files.createDirectories(path);
+    } catch (IOException e) {
+      log.error(e.getMessage());
```

Comment:
This logging pattern doesn't meet the requirements because it should have a descriptive error message and the caught throwable. 
It may be like: `log.error("Error creating parent directories for path", e);`